.gitignore

```diff
@@ -12,3 +12,4 @@ release.properties
 /.settings/
 /.classpath
 /.project
+/nbproject/
```

Comment:
Missing newline at the end of the file (for [Git](https://thoughtbot.com/blog/no-newline-at-end-of-file))

Reply:
Sorry about the .gitignore. Don't know what happened with that.  I didn't realize it got picked up by my commit.